### PR TITLE
additional k8s presentation proposal

### PR DESCRIPTION
**Pull request by lotharschulz**
(Agiles) IT Change-, Release- and Deployment Management: "Continuous Delivery to Kubernetes – Deployment Interfaces"

### CODE REVIEW

**Comment by hjacobs** [comment]
I think your PR makes sense. I would not add the other mentioned presentations (the first is too short, the other is not about Zalando's Kubernetes setup).

**Comment by hjacobs** [comment]
:+1:

**Review by hjacobs — APPROVED** [review]

File: docs/admin-guide/public-presentations.rst [hunk 1]
```diff
@@ -6,7 +6,8 @@ Public Presentations
 
 * `KubeCon Copenhagen 2018: Continuously Deliver your Kubernetes Infrastructure <https://www.youtube.com/watch?v=1xHmCrd8Qn8>`_
 * `Zalando Dortmund Meetup: Kubernetes Deployments: A “Hands-Off” Approach <https://www.youtube.com/watch?v=e0DruWvY-ME>`_
-* `KubeCon Austin 2017: Kubernetes on AWS: Practices & Opinions <https://www.youtube.com/watch?v=gegaGA7Ek9E>`_
+* `(Agiles) IT Change-, Release- and Deployment Management: Continuous Delivery to Kubernetes – Deployment Interfaces <https://www.slideshare.net/LotharSchulz1/continuous-delivery-to-kubernetes-deployment-interfaces>`_
+* `KubeCon Austin 2017: Kubernetes on AWS: Practichttps://www.slideshare.net/LotharSchulz1/continuous-delivery-to-kubernetes-deployment-interfaceses & Opinions <https://www.youtube.com/watch?v=gegaGA7Ek9E>`_
```
Comment (mikkeloscar) [inline docs/admin-guide/public-presentations.rst]: link looks broken `Practichttps://`

**Comment by hjacobs** [comment]
:-1:

**Comment by mikkeloscar** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**